pegasus/migrations/096_drop_unique_constraint_from_forms_source_id_index.rb

```diff
@@ -0,0 +1,22 @@
+Sequel.migration do
+  up do
+    # For some reason this unique index only exists on production, but not staging or test.
+    # Conditionally drop it if it exists.
+    if self.indexes(:forms)[:unique_forms_source_id]
+      alter_table(:forms) do
+        drop_index :source_id, name: :unique_forms_source_id
+      end
+    end
+
+    # Replace with a unique index on source_id and kind (rather than just source_id).
+    alter_table(:forms) do
+      add_index [:source_id, :kind], unique: true
```

Comment:
It can be unique for a given kind of form. The problem is when both [PdWorkshopSurvey](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/forms/pd_workshop_survey.rb#L91) and [PdWorkshopMaterials](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/forms/pd_workshop_materials.rb#L22) both refer to the same enrollmentId. The latter (materials) is unable to save.